packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -413,24 +425,118 @@ contract Deploy is Deployer {
     //              Non-Proxied Deployment Functions              //
     ////////////////////////////////////////////////////////////////
 
-    /// @notice Deploy the Safe
-    function deploySafe() public broadcast returns (address addr_) {
-        console.log("Deploying Safe");
+    /// @notice Deploy the SystemOwnerSafe
+    function deploySystemOwnerSafe() public broadcast returns (address addr_) {
+        console.log("Deploying System Owner Safe");
         (SafeProxyFactory safeProxyFactory, Safe safeSingleton) = _getSafeFactory();
 
         address[] memory signers = new address[](1);
         signers[0] = msg.sender;
 
-        bytes memory initData = abi.encodeWithSelector(
-            Safe.setup.selector, signers, 1, address(0), hex"", address(0), address(0), 0, address(0)
-        );
+        bytes memory initData =
+            abi.encodeCall(Safe.setup, (signers, 1, address(0), hex"", address(0), address(0), 0, payable(address(0))));
         address safe = address(safeProxyFactory.createProxyWithNonce(address(safeSingleton), initData, block.timestamp));
 
         save("SystemOwnerSafe", address(safe));
         console.log("New SystemOwnerSafe deployed at %s", address(safe));
         addr_ = safe;
     }
 
+    /// @notice Deploy a LivenessGuard for use on the Security Council Safe.
+    ///         Note this function does not have the broadcast modifier.
+    function deployLivenessGuard() public returns (address guard_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        guard_ = address(new LivenessGuard(councilSafe));
+
+        save("LivenessGuard", address(guard_));
+        console.log("New LivenessGuard deployed at %s", address(guard_));
+    }
+
+    /// @notice Deploy a LivenessModule for use on the Security Council Safe
+    ///         Note this function does not have the broadcast modifier.
+    function deployLivenessModule() public returns (address module_) {
+        Safe councilSafe = Safe(payable(mustGetAddress("SecurityCouncilSafe")));
+        address fallbackOwner = mustGetAddress("SystemOwnerSafe");
```

Comment:
Why is this role called "SystemOwner" Safe? i.e. what role does system owner correspond to, the system config owner? Presumably it's not the proxy admin owner since this is intended to be the foundation only.

How will this work in prod where we have two foundation addreses? We currently have:
- 0x847B5c174615B1B7fDF770882256e2D3E95b9D92 is the one on the 2/2 safe
- 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A is the current guardian